lib/activerecord-import/import.rb

```diff
@@ -67,7 +67,7 @@ def import(*args, &block)
 
     # supports empty array
     elsif args.last.is_a?( Array ) && args.last.empty?
-      return ActiveRecord::Import::Result.new([], 0, [])
+      return ActiveRecord::Import::Result.new([], 0, []) if args.last.empty?
```

Comment:
these seems to been from the file itself. not something I added... it's from my forked file. so it's probably not needed.